src/style.css

```diff
@@ -1,3 +1,68 @@
 body {
   margin: 0;
+
+  --selected-color: #00acdc;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  min-width: 1024px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.header__logo__img {
```

Comment:
Utilize SCSS nesting with & symbol as it can make the stylesheet more concise and maintainable.